pkg/cmd/cli/describe/printer.go

```diff
@@ -138,6 +140,11 @@ func NewHumanReadablePrinter(noHeaders, withNamespace, wide bool, showAll bool,
 	return p
 }
 
+func NewHumanReadablePrinterWithResourceAliases(resourceAliases map[string]string, noHeaders, withNamespace, wide bool, showAll bool, showLabels bool, absoluteTimestamps bool, columnLabels []string) *kctl.HumanReadablePrinter {
+	resourceShortForms = resourceAliases
```

Comment:
This should be a field of the printer, not mutate the global state (that global variable `resourceShortForms` above).

Calling the function `resourceShortFormFor` before a call to this `NewHumanReadablePrinterWithResourceAliases` will range over an empty map, while after the call it will range over whatever was the last thing passed in as argument.

And if these functions would be called from different goroutines (I believe they will be not, in the CLI code base) there's a data race.